src/fix_nh.h

```diff
@@ -139,6 +139,27 @@ class FixNH : public Fix {
   double compute_strain_energy();
   void compute_press_target();
   void nh_omega_dot();
+
+  // Implementation of CauchyStat
+  double H0[3][3];               //shape matrix for the undeformed cell
+  double h_old[6];               //previous time step shape matrix for the undeformed cell
+  double invH0[3][3];            //inverse of H0;
+  double CSvol0;                 //volume of undeformed cell
+  double setPK[3][3];            //current set values of the PK stress (this is modified until the cauchy stress converges)
+  static double setPKinit[6];    // initialization value of setPK for continuation runs
```

Comment:
Please explain why `setPK[][]`, `restartPK` and `restart_stored` have to be static class members. It looks to me, as if the `static` qualifier can be removed, if those variables are initialized in the constructor.

Reply:
setPK is not static, I think you mean setPKinit and the others (restartPK and restart_stored).  These three are static because they need to be kept up-to-date at the end of a NPT run so that they are known at the start of any subsequent NPT runs.  If there is a cleaner cpp way to do this, please advise - I am a cpp amateur.